cmake/CMakeLists.txt

```diff
@@ -490,10 +492,14 @@ endif()
 get_filename_component(libTKernelExt ${libTKernel} EXT)
 if("${libTKernelExt}" STREQUAL ".a")
     find_package(Threads)
-    if (NOT APPLE)
-        set(LIB_RT "rt")
+    # TODO why OPENCASCADE_LIBRARIES is repeated three times below?
```

Comment:
This is to resolve cyclic dependencies in gcc/ld. Better way seems to use `--start-group ... --end-group`, but didn't want to spend effort on it.